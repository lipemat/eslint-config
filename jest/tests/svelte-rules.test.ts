import jestRunnerEslint from '../../jest/helpers/jest-runner-eslint';

describe( 'Svelte rules are enabled', () => {
	test( 'svelte/no-dupe-style-properties', async () => {
		expect( await jestRunnerEslint( 'failing/svelte/no-dupe-style-properties' ) ).toMatchSnapshot();
	} );
} );
