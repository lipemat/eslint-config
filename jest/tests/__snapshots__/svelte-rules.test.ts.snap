// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svelte rules are enabled svelte/no-dupe-style-properties 1`] = `
"FAIL jest/fixtures/failing-svelte/no-dupe-style-properties/CircleProgress.svelte
  × null (mocked ms)


E:\\SVN\\the-boilerplate\\packages\\eslint-config\\jest\\fixtures\\failing-svelte\\no-dupe-style-properties\\CircleProgress.svelte
  19:0  error  Parsing error: You can only have one top-level <style> tag per component

✖ 1 problem (1 error, 0 warnings)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;
